src/api.js

```diff
@@ -1,23 +1,28 @@
-import { examRequiresAccessToken, store } from './data';
+import { useDispatch, useSelector } from 'react-redux';
+import { examRequiresAccessToken } from './data';
+
+export const useIsExam = () => {
```

Comment:
That's a good question. I agree that `isExam` is a better name than `useIsExam`, but because I've turned an ordinary JavaScript function into a React hook, and because [hooks names always start with use](https://react.dev/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use), I have to prefix the function name with `use`.